templates/zrobot/jumpscale9.toml

```diff
@@ -0,0 +1,38 @@
+[myconfig]
```

Comment:
Why do we need this file? shoudln't it be already part of the docker image you use ?